lib/index.js

```diff
@@ -113,7 +113,8 @@ exports.init = function(options, callback) {
     }
     
     var opt = {
-        depth: options.depth
+        depth: options.depth,
+        dev: options.include.indexOf('devDependencies') != -1,
```

Comment:
does `dev`: actually do anything?  maybe I missed it but I don't think its read anywhere